crates/sui-types/src/crypto.rs

```diff
@@ -1140,6 +1142,40 @@ pub struct AuthorityQuorumSignInfo<const STRONG_THRESHOLD: bool> {
 pub type AuthorityStrongQuorumSignInfo = AuthorityQuorumSignInfo<true>;
 pub type AuthorityWeakQuorumSignInfo = AuthorityQuorumSignInfo<false>;
 
+// Variant of [AuthorityStrongQuorumSignInfo] but with a serialized signature, to be used in
+// external APIs.
+#[serde_as]
+#[derive(Clone, Debug, Serialize, Deserialize, JsonSchema)]
+pub struct SuiAuthorityStrongQuorumSignInfo {
+    pub epoch: EpochId,
+    pub signature: AggregateAuthoritySignatureAsBytes,
```

Comment:
if we use this, is the `AggrAuthSignature` still needed?
```
    #[serde_as(as = "AggrAuthSignature")]
    pub signature: AggregateAuthoritySignature,
```

Reply:
I think so, but let's keep that for the follow up clean ups we want to do for all serde related code.

Reply:
The problem is that our RUST SDK clients have access to this type, and we want to decouple their code from our internal types (that's also why we have all the duplicated types in sui-json-rpc-types).
I don't think we can do that just with serde, but maybe a smart macro can do that somehow? I couldn't find a nice solution.